### DISCUSSION TITLE
Best way to debug github action?

### DISCUSSION THREAD

**Discussion by acomito** [cypress-io/cypress#22229] [Questions and Help]
When I run my test suite locally with cypress run against a docker setup of my app, everything passes 100% like 98% of the time

When I run it in a github action, using docker and using the same DB backing the instance, it passes 100% of the suite like 45% of the time... the only big difference that stands out to to me is that locally I run `cypress run` and in the github action, I'm using the `cypress-io/github-action@v4`

I'm not that familiar with github actions or cypress-- wondering what is the best way to debug this. Seems like it's often a timeout related issue? Do I need to bump up the juice for my github actions?

**Comment by jjhelguero** [comment]
CI resources are limited and can lead to command timeouts. One way to combat flakiness is sprinkling intercepts and assertions throughout your test and possibly breaking down longer tests into small spec files.
